crates/packager/src/lib.rs

```diff
@@ -154,7 +154,10 @@ pub fn sign_outputs(
                 let zip = path.with_extension(extension);
                 let dest_file = util::create_file(&zip)?;
                 let gzip_encoder = libflate::gzip::Encoder::new(dest_file)?;
-                util::create_tar_from_dir(path, gzip_encoder)?;
+                let writer = util::create_tar_from_dir(path, gzip_encoder)?;
```

Comment:
I checked this function, why isn't it using tar's append methods? seems like a custom implementation instead of something like https://docs.rs/tar/latest/tar/struct.Builder.html#method.append_dir_all

Reply:
not sure what you mean by that, it is using `append_data` to in order to set the `uid` and `gid`